packages/mux-player/src/index.ts

```diff
@@ -153,8 +154,10 @@ function getProps(el: MuxPlayerElement, state?: any): MuxTemplateProps {
     customDomain: el.getAttribute(MuxVideoAttributes.CUSTOM_DOMAIN) ?? undefined,
     title: el.getAttribute(PlayerAttributes.TITLE),
     novolumepref: el.hasAttribute(PlayerAttributes.NO_VOLUME_PREF),
-    extraPlaylistParams: { redundant_streams: true },
     ...state,
+    // NOTE: since the attribute value is used as the "source of truth" for the property getter,
+    // moving this below the `...state` spread so it resolves to the default value when unset (CJP)
+    extraPlaylistParams: el.extraPlaylistParams,
```

Comment:
We should consider this convention for all of the acrobatics we've had with our generic state update from attr values in `attributeChangedCallback()`. tl;dr - if our mux-player prop value is always derived from the corresponding mux-player attr value, we should move those below our `state` spread. (cc @luwes)